src/core/stdc/string.d

```diff
@@ -29,80 +29,80 @@ nothrow:
 @nogc:
 
 ///
-pure void* memchr(in void* s, int c, size_t n);
+pure void* memchr(return const void* s, int c, size_t n);
 ///
-pure int   memcmp(in void* s1, in void* s2, size_t n);
+pure int   memcmp(scope const void* s1, scope const void* s2, size_t n);
```

Comment:
So `in` != `scope const`. If I remember correctly, you said that you did that because it would break too much code.
Do you expect that to change? I ask because when I write code I need to know the meaning of `in` and so that my code would be compatible with DIP1000.